doc/api/errors.md

```diff
@@ -1226,6 +1226,11 @@ reached.
 An attempt was made to initiate a new push stream from within a push stream.
 Nested push streams are not permitted.
 
+<a id="ERR_HTTP2_NO_MEM"></a>
+### `ERR_HTTP2_NO_MEM`
```

Comment:
Not for this PR, but I wonder if it might serve users better to have a single generic memory exhaustion error and combine this and ERR_WORKER_OUT_OF_MEMORY (and prevent a proliferation of other similar errors in the future).

Reply:
> a single generic memory exhaustion error

This error is for `http2` only and simple enough, maybe it's a little over engineered to do this ?

Reply:
What's wrong with `ENOMEM`?

Reply:
Maybe it's more specific to `http2`.